test/scenarios.py

```diff
@@ -7875,4 +7875,52 @@
             ),
         },
     ),
+    Scenario(
+        "scenario39",
+        "Produce a situation in which the receiver has stalled transfer, but sender does not have it anymore. Expect receiver to check the transfer state eventually",
+        {
+            "ren": ActionList(
+                [
+                    action.Start("172.20.0.5"),
+                    action.NewTransfer("172.20.0.15", ["/tmp/testfile-small"]),
+                    action.Wait(
+                        event.Queued(
+                            0,
+                            {
+                                event.File(
+                                    FILES["testfile-small"].id,
+                                    "testfile-small",
+                                    1048576,
+                                ),
+                            },
+                        )
+                    ),
+                    action.SleepMs(200),
+                    action.Stop(),
+                    action.Start("172.20.0.5"),
+                    action.Sleep(61),  # 60s is a check request interval
```

Comment:
I had a hard time understanding how it stalls until I realized that we use :memory: for SQLite. Can we add explicit different storage paths so it would be apparent?

Reply:
I guess using on-disk storage for this test case is overkill but I can add a comment explaining

Reply:
done